gems/aws-sdk-core/spec/aws/log/param_filter_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../../spec_helper'
+
+module Aws
+  module Log
+    describe ParamFilter do
+      describe '#filter' do
```

Comment:
Does it make sense to add other cases while you're here? One for struct/array? I won't block you on that, but it'd be helpful. :)

Reply:
I'd be happy to - the more test cases the merrier :)

Reply:
Done!